lib/auth/clt.go

```diff
@@ -1312,7 +1312,22 @@ func (c *Client) UpsertPassword(user string, password []byte) error {
 	return nil
 }
 
-// UpsertUser user updates or inserts user entry
+// CreateUser inserts a new entry in a backend.
+func (c *Client) CreateUser(ctx context.Context, user services.User) error {
+	clt, err := c.grpc()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	_, err = clt.CreateUser(ctx, user.(*services.UserV2))
```

Comment:
This should err instead of panic.  Ex:
```golang
v2, ok := user.(*services.UserV2)
if !ok {
    return trace.BadParameter("unsupported user type %T",user)
}
// ...
```

Reply:
revised: https://github.com/gravitational/teleport/pull/3602/commits/cdeccd688f313c252f785b0919b4f1d786e1a8d7